core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -4376,6 +4411,8 @@ protected function parentView()
 
 			for ($i=0, $c=\count($row); $i<$c; $i++)
 			{
+				$this->denyAccessUnlessGranted(ContaoCorePermissions::DCA_VIEW, new DataContainerSubject($this->strTable, $row[$i]['id']));
```

Comment:
this means if I want to limit an element from being visible, I must not use this permission but also apply a `filter` or similar to the DCA, right? Otherwise we should catch the exception and not list the item.

Reply:
Yes, this is only a security measure here.